packages/sources/tradermade/README.md

```diff
@@ -1,32 +1,94 @@
 # Chainlink External Adapter for Tradermade
 
+## Endpoints
+
+- `live` and `forex`. Use `forex` when querying for currency pairs. WebSockets can be enabled when fetching data from the `forex` endpoint but not the `live` endpoint. Adapter defaults to the `live` endpoint.
+
```

Comment:
Can we update this README to follow the format of how we usually document adapters with different endpoints? That would give us some more room to explain each endpoint in their own respective sections